### PR TITLE
Documented option --warnings-only-to-file

### PR DESCRIPTION
**Pull request by ccadar**

### CODE REVIEW

File: docs/options.md [hunk 1]
```diff
@@ -21,7 +21,24 @@ $ klee [klee-options] <program.bc> [program-options]
 The general form of a KLEE command-line is first the arguments for KLEE itself (`[klee-options]`), then the LLVM bitcode file to execute (`program.bc`), and then any arguments to pass to the application (`[program-options]`).
 In particular, the KLEE option `-posix-runtime` enables the use of the symbolic environment options as part of the program's options.
 
-To get a complete list of KLEE's command-line options run: `klee --help`. The remainder of this page illustrate KLEE's main command-line options.
+To get a complete list of KLEE's command-line options run: `klee
```
Comment (andreamattavelli) [inline docs/options.md]: Please do not force hard wrap of lines.

File: docs/options.md [hunk 2]
```diff
@@ -21,7 +21,24 @@ $ klee [klee-options] <program.bc> [program-options]
...
+--help`. The remainder of this page illustrate KLEE's main
+command-line options.
+
+
+## KLEE Output
+
+The files generated by KLEE are discussed [here]({{site.baseurl}}/docs/files).
+
+By default KLEE outputs warnings both on screen and in the file
+`klee/lastwarnings.txt`.  To output the warnings only to the file, and
```
Comment (andreamattavelli) [inline docs/options.md]: Should be `klee_last/warnings.txt`.

**Comment by ccadar** [comment]
Thanks, fixed

**Comment by andreamattavelli** [comment]
Thanks, merged

**Pull request merged**